spec/seeders/basic_data/project_role_seeder_spec.rb

```diff
@@ -103,42 +105,15 @@
     end
   end
 
-  context 'with some global roles defined' do
-    let(:data_hash) do
-      YAML.load <<~SEEDING_DATA_YAML
-        roles:
-        - reference: :role_staff_manager
-          name: Staff manager
-          global: true
-          permissions:
-          - :hire_people
-          - :give_feedback
-      SEEDING_DATA_YAML
-    end
-
-    it 'creates the corresponding global roles with the given attributes' do
-      expect(GlobalRole.count).to eq(1)
-      expect(GlobalRole.find_by(name: 'Staff manager')).to have_attributes(
-        builtin: Role::NON_BUILTIN,
-        permissions: %i[hire_people give_feedback]
-      )
-    end
-
-    it 'references the role in the seed data' do
-      role = GlobalRole.find_by(name: 'Staff manager')
-      expect(seed_data.find_reference(:role_staff_manager)).to eq(role)
-    end
-  end
-
   context 'with permissions: :all_assignable_permissions' do
     let(:data_hash) do
       YAML.load <<~SEEDING_DATA_YAML
         roles:
-        - reference: :role_project_admin
-          name: Project admin
-          global: true
```

Comment:
@cbliard Following up on the `assignable_permissions` comment. This was intended to be a global role, but based on the spec it was expected to be assigned all `assignable_permissions` for the `Role` type.

Now that I've split these specs into `project_role_seeder_spec` and `global_role_seeder_spec` I've added an expectation on the `global_role_seeder_spec` that the assigned roles to the created role, only gets the `assignable_permissions` for `GlobalRole` types.

Reply:
I think I did a mistake originally here. As it is the "Project admin" role, it should not have been `global: true`.
Thanks for having fixed it!